src/lib.rs

```diff
@@ -175,6 +185,11 @@ pub struct ImportLibraryGenerator {
     version: Option<(u8, u8)>,
     /// Python interpreter implementation
     implementation: PythonImplementation,
+    /// Optional Python ABI tag
+    ///
+    /// For example, `"t"` stands for the free-threaded CPython v3.13 build
+    /// aka CPython `3.13t`.
+    abi_tag: Option<String>,
```

Comment:
I think the more normal name I've seen for these might be "abiflags". That's at least what they're called in `sysconfig` and `maturin`. I think the full "abi tag" would be `cp313t` or `abi3` based upon [the packaging spec](https://packaging.python.org/en/latest/specifications/platform-compatibility-tags/#platform-compatibility-tags).

```suggestion
    abiflags: Option<String>,
```

Reply:
Applied.